.snyk

```diff
@@ -69,12 +69,24 @@ ignore:
         reason: >-
           No remediation path available, and eval function is not used. Upstream uses pysolr, which will resolve 
           the issue.
-        expires: 2020-11-17T06:00:00.000Z
+        expires: 2020-12-23T06:00:00.000Z
   SNYK-PYTHON-URLLIB3-1014645:
     - '*':
         reason: >-
           No remediation path available for CKAN2.3 (not compatible with urllib3 > 1.24). Need to resolve moving to
           CKAN2.8 for catalog (https://github.com/GSA/datagov-ckan-multi/issues/298). `method` parameter is not
           utilized for any user input value, risk is acceptable.
-        expires: 2020-11-17T06:00:00.000Z
+        expires: 2020-12-23T06:00:00.000Z
+  SNYK-PYTHON-RSA-1038401:
+    - '*':
+        reason: >-
+          No remediation path available. Catalog-next does not use rsa, which will resolve 
```

Comment:
I don't think this is a valid exception. I don't think catalog-next is guaranteed to ship within 30 days but more importantly, I don't think it's true that catalog-next won't also have this problem. It looks like this is introduced from ckanext-saml2 which hasn't been added as a dependency to catalog-next, yet. The extension _will_ be added once the authentication story is done and I think it will have the same problem.

Since this basically bumps us out of the white noise threshold, let's add the 30 day exception but open an issue to find a mitigation and get started on that asap. Let's look into these areas:
1. Is RSA/M2Crypto working on a fix within the next 30 days?
2. Is catalog avoiding the vulnerable code paths? 
3. Is catalog only using the vulnerable code paths to parse data/certificates we control?
4. Is there a workaround to mitigate Bleichenbacher timing attacks?

The exception here would then become "We are working on a fix <link-to-issue>". A "Yes" to any of these questions ☝️  would clarify/resolve the exception.

Reply:
👍 [Issue created](https://github.com/GSA/datagov-deploy/issues/2472), and reason updated.